XRTK-Core/Packages/com.xrtk.core/Editor/BuildAndDeploy/BuildDeployWindow.cs

```diff
@@ -525,16 +524,16 @@ private void AppxBuildGUI()
 
             var prevFieldWidth = EditorGUIUtility.fieldWidth;
 
-            EditorGUIUtility.fieldWidth = 120;
-            var appIconAssetObject = AssetDatabase.LoadAssetAtPath<GltfAsset>(UwpBuildDeployPreferences.MixedRealityAppIconPath);
+            //EditorGUIUtility.fieldWidth = 120;
+            //var appIconAssetObject = AssetDatabase.LoadAssetAtPath<GltfAsset>(UwpBuildDeployPreferences.MixedRealityAppIconPath);
 
```

Comment:
Let's just go ahead and remove the lines altogether. I'm rewriting this anyway and don't want conflicts.

Reply:
Darn, I thought I had deleted everything.  Good spot.